File: Kyu 7 Exes and Ohs/solution.cs
using System;

public class Solution
{
    public static bool XO(string input)
    {
        var xCount = 0;
        var oCount = 0;
        input = input.ToLower();
        foreach (var c in input)
        {
            if (c == 'x')
            {
                xCount++;
            }
            else if (c == 'o')
            {
                oCount++;
            }
        }
        return xCount == oCount;
    }

    public static void Main(string[] args)
    {
        var testCases = new string[] { "xo", "xxOo", "xxxm", "Oo", "ooom", "xxxoo" };
        foreach (var testCase in testCases)
        {
            Console.WriteLine(XO(testCase));
        }
    }
}

File: Kyu 7 Exes and Ohs/solution.cpp
#include <iostream>
#include <string>

bool XO(std::string s)
{
    int x = 0, o = 0;
    for (int i = 0; i < s.size(); i++)
    {
        char c = std::tolower(s[i]);
        if (c == 'x')
            x++;
        else if (c == 'o')
            o++;
    }
    return x == o;
}

int main()
{
    std::string testCases[] = {"xo", "xxOo", "xxxm", "Oo", "ooom", "xxxoo"};
    for (std::string s : testCases)
    {
        std::cout << s << ": " << (XO(s) ? "True" : "False") << std::endl;
    }
}